src/components/DirectorHeadshot/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import path from 'path';
+
+import ResponsiveImage from 'components/ResponsiveImage';
+
+const DirectorHeadshot = ({ photo }) => {
+  const filename = photo.filename;
+  const photoExtension = path.extname(filename);
+  const photoBasename = path.basename(
+    filename,
+    photoExtension,
+  );
+  const filepath = `${process.env.CMS_MEDIA}/images/${photoBasename}`;
+
+  return (
+    <div className="coa-DepartmentPage__directorcard-headshot">
+      <ResponsiveImage
+        className="coa-DirectorHeadshotImage"
+        filename={filepath}
+        defaultWidth="width-640"
+        widths={[]}
```

Comment:
What happens if this is unset instead of an empty array?

Reply:
That was intentional. I think the page broke or the image didn't show up if widths (and by extension srcSet) was undefined.